framework/api_vulkan_sample.cpp

```diff
@@ -1,3 +1,4 @@
+/* Copyright (c) 2019-2024, Sascha Willems
```

Comment:
You've doubled the Copyright comment here, resulting in doxygen failing.

Reply:
Thanks 👍🏻  Most probably happened while trying to merge